must-gather/collection-scripts/gather_common_ceph_resources

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Expect base collection path as an argument
+BASE_COLLECTION_PATH=$1
+
+# Expect time option as an argument
+SINCE_TIME=$2
+
+# Use PWD as base path if no argument is passed
+if [ "${BASE_COLLECTION_PATH}" = "" ]; then
+    BASE_COLLECTION_PATH=$(pwd)
+fi
+
+CEPH_COLLECTION_PATH="${BASE_COLLECTION_PATH}/ceph"
+# pre-install-ceph-resources.sh
+
+# External Ceph resources
+external_ceph_resources=()
+external_ceph_resources+=(cephobjectstores)
+external_ceph_resources+=(cephobjectstoreusers)
+external_ceph_resources+=(cephclusters)
```

Comment:
Probably rename these to `common_*`

Reply:
I will raise a separate pr for refactoring

Reply:
@rajatsing you are introducing this change in this PR so you can  update these changes in this PR. I don’t think there should need for separate PR. Thoughts?